.github/workflows/prod-cd.yaml

```diff
@@ -0,0 +1,63 @@
+name: Production CD
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: [self-hosted, rpi, prod]
+    environment: yacs-prod
```

Comment:
lets rename this to `yacs-rpi-prod` since rpi specific.
besides that, awesome!

Reply:
Yep changed